src/Controller/InventoryController.java

```diff
@@ -58,29 +58,35 @@ public void setArtworkInventory() {
         artworkControllerInstance.showArtworks();
         System.out.println("Select Artwork: ");
         String artworkName = scanner.next();
-        ArtWork artWork = artworkControllerInstance.getArtworkList().get(artworkName);
-        System.out.println("Select destination: ");
-        showInventories();
-        String artworkLocation = scanner.next();
-        if (inventoryList.containsKey(artworkLocation)) {
-            if (rentalControllerInstance.getInstituteList().containsKey(artWork.getInventoryLocation()) ||
-                    expositionControllerInstance.getExpoList().containsKey(artWork.getInventoryLocation())) {
-                System.out.println(artworkName + " is located in: " + artWork.getInventoryLocation());
-                System.out.println("Do you wish to continue? Y/N");
-                String useryn = scanner.next();
-                if (useryn.equalsIgnoreCase("y")) {
-                    if (rentalControllerInstance.onActiveRent(artWork) || expositionControllerInstance.onActiveExpo(artWork)) {
-                        System.out.println("Artwork can not be put into inventory \n Check Rental or Expo status");
+        LinkedHashMap<String, ArtWork> artList = artworkControllerInstance.getArtworkList();
```

Comment:
i really dont like the fact we continually ask for the getArtworkList and keep saving it. I think for each controller class that is not artwork (that one already has it), you need to make sure to add a class attribute where you save artworkControllerInstance.getArtworkList() and that way you can use it across all your class methods without having to keep consulting it over and over on each method